src/invariant/test/OrderBookIsNotCrossedTests.cpp

```diff
@@ -0,0 +1,333 @@
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
see me other comment: I'm pretty sure we can always test that the invariant works, regardless of fuzzing enabled or not

Reply:
(otherwise this test case will never be run anywhere)

Reply:
wow that was a jar jar binks comment from my part.
Meesa resolve this comment now!